torchvision/prototype/features/_bounding_box.py

```diff
@@ -7,6 +7,7 @@
 from torchvision.transforms import InterpolationMode  # TODO: this needs to be moved out of transforms
 
 from ._feature import _Feature
+from ._utils import FillType
```

Comment:
Centralized this one in a `_utils.py` file to avoid instances of misalignment with `functional`.